README.md

```diff
@@ -1,11 +1,47 @@
 Yoti Golang SDK
 =============
 
-Welcome to the Yoti Golang SDK. This repo contains the tools you need to quickly integrate your golang back-end with Yoti, so that your users can share their identity details with your application in a secure and trusted way.    
+Welcome to the Yoti Golang SDK. This repo contains the tools and step by step instructions you need to quickly integrate your golang back-end with Yoti, so that your users can share their identity details with your application in a secure and trusted way.
 
-## An architectural view
-To integrate your application with Yoti, your back-end must expose a GET endpoint that Yoti will use to forward tokens.
-The endpoint can be configured in Yoti Dashboard when you create/update your application.
+Feel free to use the example above: [yoticlient.go](https://github.com/getyoti/go/blob/0af652c98a90a8a5b75e933828b95a77eb7c1052/yoticlient.go) to get started. You can also clone/download the example on your desktop to replicate our SDK at ease!
```

Comment:
On second thought, you can remove this paragraph. I will add it to my pull request, so you don't have to wait for it. 👏